tests/validation/tests/v3_api/test_secrets.py

```diff
@@ -308,6 +313,7 @@ def test_rbac_secret_edit_cluster_member(remove_resource):
     assert e.value.error.code == 'NotFound'
 
     remove_resource(project)
+    remove_resource(ownerproject)
```

Comment:
Move all the remove_resource() before any asserts

Reply:
Done